### ISSUE TITLE
Typo?

### ISSUE THREAD

**Issue by erikkemperman** [1j01/process-tree#1]
I suspect a typo in [index.coffee](https://github.com/1j01/process-tree/blob/master/index.coffee#L41) -- the `proc` variable is invalid here. Should it just have been `info`?

I am seeing this in `atom-mocha-test-runner` which has switched its dependency from `ps-tree` to `process-tree` recently. Ref [#58](https://github.com/TabDigital/atom-mocha-test-runner/issues/58).

**Comment by 1j01** [comment]
Hm, yeah, looks like I probably renamed `proc` to `info` manually and missed an occurrence.
I'll fix this now, although I'm not sure if it's falling back to `cmd` if it'll actually work.

**Comment by 1j01** [comment]
Published `process-tree@1.0.1`

It's supposed to find "Name" from `wmic`, or "comm" from `ps`, though, so it might be missing a column or something when parsing the output. Or maybe `ps` gives a different column name on OSX than on Linux.

Can you run `ps` to see what the output looks like on OSX?

**Comment by erikkemperman** [comment]
Thanks for the quick response! I ran into this on linux, actually, but can run `ps` in osx when I get home later.

**Comment by 1j01** [comment]
Well, output from wherever it's failing would be helpful.

**Comment by erikkemperman** [comment]
``` sh
PID TTY           TIME CMD
707 ttys000    0:00.05 -bash
```

I suspect this is the same as linux, but I'll reboot into linux now and confirm.

**Comment by erikkemperman** [comment]
The above was osx, this is linux;

``` sh
  PID TTY          TIME CMD
 2361 pts/2    00:00:00 bash
 2373 pts/2    00:00:00 ps
```

Thanks again!

**Comment by erikkemperman** [comment]
Hmm I just did `ps` without any args earlier, but now I noticed that with the flags you're passing the column headers are different:

``` sh
PPID   PID COMMAND
```

Be right back after booting into OSX again.

**Comment by erikkemperman** [comment]
OSX column headers with flags `-A -o ppid,pid,comm`

``` sh
PPID   PID COMM
```

Sorry for spamming, should have checked for args to begin with.

**Comment by 1j01** [comment]
Alright, so I probably just need to make it case-insensitive.

**Comment by 1j01** [comment]
Published `process-tree@1.0.2`
